kernel/core/plotting/plotting.cpp

```diff
@@ -4349,9 +4253,13 @@ void Plot::createDataLegends()
 /////////////////////////////////////////////////
 string Plot::constructDataLegendElement(string& sColumnIndices, const string& sTableName, bool useBrackets)
 {
+#warning NOTE (erik.haenel#1#): This function is now dead. Remove it.

```

Comment:
So should it be removed? Otherwise note when this has to be executed.

Reply:
Good point. Will remove it.